automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookRKernelSpec.scala

```diff
@@ -151,7 +151,7 @@ class NotebookRKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
           notebookPage
             .executeCell("Sys.getenv('WORKSPACE_NAMESPACE')")
             .get shouldBe s"'${runtimeFixture.runtime.googleProject.value}'"
-          notebookPage.executeCell("Sys.getenv('WORKSPACE_NAME')").get shouldBe "'Untitled Folder'"
```

Comment:
not sure why it becomes `[]`, but the variable is populated properly when I tried out in fiab

Reply:
So it gets set from here: https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/scripts/kernel/kernel_bootstrap.sh#L16

That's why it was `Untitled Folder` in the test, since that is the name of the parent dir. 

Is the `kernel_bootstrap.sh` script running still? Might be worth manually checking all the EVs from a notebook.

Reply:
Yea I did...it looks correct to me...there're some extra gpu stuff, but all the WORKSPACE_NAMESPACE, WORKSPACE_BUCKET, WORKSPACE_NAME etc are all there
```
CUDNN_VERSION=8.0.4.30
LC_ALL=en_US.UTF-8
LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64
_=/usr/bin/env
LANG=C.UTF-8
NOTEBOOKS_DIR=/home/jupyter/notebooks
HOSTNAME=saturn-541442d5-e903-4578-a5df-77bf42d85129
OWNER_EMAIL=hermione.owner@test.firecloud.org
NVIDIA_VISIBLE_DEVICES=all
USER=jupyter
PAGER=cat
NCCL_VERSION=2.7.8
WELDER_ENABLED=true
JPY_PARENT_PID=28
R_LIBS=/home/jupyter/notebooks/packages
PWD=/home/jupyter/notebooks/ws/edit
HOME=/home/jupyter
DL_ANACONDA_HOME=/opt/conda
CLICOLOR=1
WELDER_USER=welder-user
JUPYTER_PORT=8000
DEBIAN_FRONTEND=noninteractive
ANACONDA_PYTHON_VERSION=3.7
LIBRARY_PATH=/usr/local/cuda/lib64/stubs
RUNTIME_NAME=saturn-541442d5-e903-4578-a5df-77bf42d85129
CONTAINER_NAME=base-cu110
TERM=xterm-color
SHELL=/bin/bash
MPLBACKEND=module://ipykernel.pylab.backend_inline
CUDA_VERSION=11.0.3
WORKSPACE_NAME=ws
NVIDIA_DRIVER_CAPABILITIES=compute,utility
SHLVL=2
PYTHONPATH=/etc/jupyter/custom:/home/jupyter/notebooks/packages
GIT_PYTHON_REFRESH=quiet
USER_INSTALL_DIR=/home/jupyter/notebooks/packages
CONTAINER_URL=gcr.io/deeplearning-platform-release/base-cu110:nightly-2021-03-28
NVIDIA_REQUIRE_CUDA=cuda>=11.0 brand=tesla,driver>=418,driver<419 brand=tesla,driver>=440,driver<441 brand=tesla,driver>=450,driver<451
GOOGLE_PROJECT=qi-billing-crypto
WORKSPACE_NAMESPACE=qi-billing-crypto
WELDER_UID=1001
JUPYTER_HOME=/etc/jupyter
PATH=/opt/conda/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/jupyter/notebooks/packages/bin:/home/jupyter/.local/bin
PIP_USER=false
PIP_TARGET=/home/jupyter/notebooks/packages
WORKSPACE_BUCKET=gs://fc-79b04e8e-b688-44e0-95d5-3a8f0c513ad0
CLUSTER_NAME=saturn-541442d5-e903-4578-a5df-77bf42d85129
GIT_PAGER=cat
```

Reply:
Hm, are you using Terra UI? I think the UI sets it:
https://github.com/DataBiosphere/terra-ui/blob/8119dfb79ef3d4c271d2c521b67f97d4ef5b70a7/src/components/NewRuntimeModal.js#L292

However since PPW is not rolled out yet I think we still want `kernel_bootstrap.sh` working (which sets the EV dynamically at kernel startup time).

So maybe a test could be:
1. create a runtime + notebook in workspace A
2. do `os.getenv('WORKSPACE_NAME')` in a notebook, verify you get workspace A
3. go to workspace B (in the same project), open new notebook
4. `os.getenv('WORKSPACE_NAME')` should return workspace B

Reply:
I was running via local terra UI....but it does seem like `WORKSPACE_NAME` isn't correct when I switch to a new workspace (it's being set as the same as the other)